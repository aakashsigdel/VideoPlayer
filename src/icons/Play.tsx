import * as React from 'react'

export const Play: React.FunctionComponent = () => (
  <svg width="24" height="24" viewBox="0 0 24 24" fill="#fff">
    <path d="M8 5v14l11-7z" />
    <path d="M0 0h24v24H0z" fill="none" />
  </svg>
)
