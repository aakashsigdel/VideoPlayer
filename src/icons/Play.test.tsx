import * as React from 'react'
import { create } from 'react-test-renderer'

import { Play } from './Play'

describe('<Play />', () => {
  it('should render correctly', () => {
    const tree = create(<Play />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
