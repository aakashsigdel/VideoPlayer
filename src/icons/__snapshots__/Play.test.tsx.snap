// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Play /> should render correctly 1`] = `
<svg
  fill="#fff"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M8 5v14l11-7z"
  />
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
</svg>
`;
